ci/test-badge.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -ex
+
+source common-setup.sh
+
+popper badge cloudlab
+
+set +e
+popper badge
+if [ $? -eq 0 ];
+then
+  exit 1
+fi
+
+popper badge errorout
+
+if [ $? -eq 0 ];
+then
+  exit 1
+fi
+
+set -e
```

Comment:
can we test that when given the proper flag/value, the command generates the badge link?